README errors

[This code snippet](https://github.com/ORNL/ReSolve/blob/develop/README.md?plain=1#L13) from the README is incorrect.

```shell
$ git clone https://code.ornl.gov/peles/resolve.git
$ mkdir build && cd build
$ cmake ../resolve
$ make
```
=>

1. wrong clone url
2. cmake .. (not ../resolve; that causes a CMAKE_MACOSX_RPATH issue)

> > 2. cmake .. (not ../resolve; that causes a CMAKE_MACOSX_RPATH issue)
> 
> How does this affect CMAKE_MACOSX_RPATH? I would think that all rpaths are set as absolute paths.

I'm not entirely sure. Maybe something to do with how it treats the CMake files depending on the current directory. `cmake ../resolve` does work from the top directory. 

```
(base) litz372@WE45172 ~/proj/ReSolve % cd build
(base) litz372@WE45172 ~/proj/ReSolve/build % cmake ../resolve
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) in CMakeLists.txt:
  cmake_minimum_required() should be called prior to this top-level project()
  call.  Please see the cmake-commands(7) manual for usage documentation of
  both commands.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is AppleClang 15.0.0.15000040
-- The CXX compiler identification is AppleClang 15.0.0.15000040
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Warning (dev) in CMakeLists.txt:
  No cmake_minimum_required command is present.  A line of code such as

    cmake_minimum_required(VERSION 3.26)

  should be added at the top of the file.  The version specified may be lower
  if you wish to support older CMake versions for this project.  For more
  information run "cmake --help-policy CMP0000".
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done (1.3s)
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   ReSolve
   resolve_backend_cpu
   resolve_logger
   resolve_matrix
   resolve_vector
   resolve_workspace

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done (0.1s)
-- Build files have been written to: /Users/litz372/proj/ReSolve/build

<<ccmake output>>
 CMAKE_BACKWARDS_COMPATIBILITY    2.4
 CMAKE_BUILD_TYPE
 CMAKE_INSTALL_PREFIX             /usr/local
 CMAKE_OSX_ARCHITECTURES
 CMAKE_OSX_DEPLOYMENT_TARGET      13.6
 CMAKE_OSX_SYSROOT                /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk
 EXECUTABLE_OUTPUT_PATH
 LIBRARY_OUTPUT_PATH
<<see how vars like RESOLVE_USE_CUDA are missing>>

(base) litz372@WE45172 ~/proj/ReSolve/build % cmake ..
CMake Error: The source "/Users/litz372/proj/ReSolve/CMakeLists.txt" does not match the source "/Users/litz372/proj/ReSolve/resolve/CMakeLists.txt" used to generate cache.  Re-run cmake with a different source directory.
(base) litz372@WE45172 ~/proj/ReSolve/build % cd ..
(base) litz372@WE45172 ~/proj/ReSolve % rm -rf build/*
zsh: sure you want to delete all 9 files in /Users/litz372/proj/ReSolve/build [yn]? y
(base) litz372@WE45172 ~/proj/ReSolve % cd build
(base) litz372@WE45172 ~/proj/ReSolve/build % cmake ..
-- The C compiler identification is AppleClang 15.0.0.15000040
-- The CXX compiler identification is AppleClang 15.0.0.15000040
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- KLU was not found.
-- Cannot find KLU, disabling SuiteSparse module ...
CMake Error at /opt/homebrew/Cellar/cmake/3.26.3/share/cmake/Modules/CMakeDetermineCUDACompiler.cmake:180 (message):
  Failed to find nvcc.

  Compiler requires the CUDA toolkit.  Please set the CUDAToolkit_ROOT
  variable.
Call Stack (most recent call first):
  CMakeLists.txt:73 (enable_language)


-- Configuring incomplete, errors occurred!
(base) litz372@WE45172 ~/proj/ReSolve/build % cd ..
(base) litz372@WE45172 ~/proj/ReSolve % rm -rf build/*
zsh: sure you want to delete all 2 files in /Users/litz372/proj/ReSolve/build [yn]? y
(base) litz372@WE45172 ~/proj/ReSolve % cmake ../resolve
-- The C compiler identification is AppleClang 15.0.0.15000040
-- The CXX compiler identification is AppleClang 15.0.0.15000040
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Library/Developer/CommandLineTools/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Library/Developer/CommandLineTools/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- KLU was not found.
-- Cannot find KLU, disabling SuiteSparse module ...
CMake Error at /opt/homebrew/Cellar/cmake/3.26.3/share/cmake/Modules/CMakeDetermineCUDACompiler.cmake:180 (message):
  Failed to find nvcc.

  Compiler requires the CUDA toolkit.  Please set the CUDAToolkit_ROOT
  variable.
Call Stack (most recent call first):
  CMakeLists.txt:73 (enable_language)


-- Configuring incomplete, errors occurred!

```

--- a/NOTICE
+++ b/NOTICE
@@ -9,12 +9,14 @@ or implied, or assumes any legal liability or responsibility for the accuracy,
 completeness, or usefulness or any information, apparatus, product, software,
 or process disclosed, or represents that its use would not infringe privately
 owned rights.
+
 Reference herein to any specific commercial product, process, or service by
 trade name, trademark, manufacturer, or otherwise does not necessarily
 constitute or imply its endorsement, recommendation, or favoring by the United
-States Government or any agency thereof, or Battelle Memorial Institute. The
-views and opinions of authors expressed herein do not necessarily state or
-reflect those of the United States Government or any agency thereof.
+States Government or any agency thereof, or UT Batelle or Battelle Memorial
+Institute. The views and opinions of authors expressed herein do not
+necessarily state or reflect those of the United States Government or any
+agency thereof.
 
                      OAK RIDGE NATIONAL LABORATORY
                               operated by

--- a/README.md
+++ b/README.md
@@ -6,8 +6,9 @@ ReSolve is a library of GPU-resident linear solver. It contains iterative and di
 
 Dependencies:
 - KLU, AMD and COLAMD libraries from SuiteSparse >= 5.0
-- CUDA >= 11.4
 - CMake >= 3.22
+- CUDA >= 11.4 (optional)
+- HIP/ROCm >= 5.6 (optional)
 
 To build it:
 ```shell

--- a/README.md
+++ b/README.md
@@ -5,15 +5,15 @@ ReSolve is a library of GPU-resident linear solver. It contains iterative and di
 ## Getting started
 
 Dependencies:
-- KLU, AMD and COLAMD libraries from SuiteSparse
+- KLU, AMD and COLAMD libraries from SuiteSparse >= 5.0
 - CUDA >= 11.4
 - CMake >= 3.22
 
 To build it:
 ```shell
-$ git clone https://code.ornl.gov/peles/resolve.git
+$ git clone https://github.com/ORNL/ReSolve.git
 $ mkdir build && cd build
-$ cmake ../resolve
+$ cmake ../ReSolve
 $ make
 ```
 
@@ -56,41 +56,16 @@ For all contributions to ReSolve please follow the [developer guidelines](CONTRI
 
 ReSolve as a library is tested every merge request via Gitlab pipelines that execute various library tests including a test of ReSolve being consumed as package within an external project as mentioned in [Using ReSolve in your own Project](#to-use-the-resolve-library-in-your-own-project)
 
-To test your own install of ReSolve simply cd into your ReSolve build directory and run 
-
+To test your own install of ReSolve simply run from your ReSolve build directory 
 ```shell
 $ make test
 ```
-
-or
-
+After you `make install` you can test your installation by running
 ```shell
-$ ctest
+$ make test_install
 ```
+from your build directory.
 
-Below is an example of what a functional ReSolve build will ouput on Passing tests
-
-```text
-Test project /people/dane678/resolve/build
-    Start 1: resolve_consume
-1/9 Test #1: resolve_consume ..................   Passed   16.51 sec
-    Start 2: klu_klu_test
-2/9 Test #2: klu_klu_test .....................   Passed    1.04 sec
-    Start 3: klu_rf_test
-3/9 Test #3: klu_rf_test ......................   Passed    1.04 sec
-    Start 4: klu_rf_fgmres_test
-4/9 Test #4: klu_rf_fgmres_test ...............   Passed    3.14 sec
-    Start 5: klu_glu_test
-5/9 Test #5: klu_glu_test .....................   Passed    1.06 sec
-    Start 6: matrix_test
-6/9 Test #6: matrix_test ......................   Passed    0.03 sec
-    Start 7: matrix_handler_test
-7/9 Test #7: matrix_handler_test ..............   Passed    0.97 sec
-    Start 8: vector_handler_test
-8/9 Test #8: vector_handler_test ..............   Passed    0.98 sec
-    Start 9: logger_test
-9/9 Test #9: logger_test ......................   Passed    0.03 sec
-```
 
 ### Important Notes
 
@@ -103,12 +78,13 @@ cmake -B build --preset cluster
 ```
 
 ## Support
-For any questions or to report a bug please submit a [GitHub issue](https://github.com/ORNL/ReSolve/issues). 
+For technical questions or to report a bug please submit a [GitHub issue](https://github.com/ORNL/ReSolve/issues).
+For non-technical issues please contact Kasia &#346;wirydowicz <kasia.swirydowicz@pnnl.gov> or Slaven Peles <peless@ornl.gov>.
 
 ## Authors and acknowledgment
-Primary authors of this project are Kasia &#346;wirydowicz <kasia.swirydowicz@pnnl.gov> and Slaven Peles <peless@ornl.gov>.
+Primary authors of this project are Kasia &#346;wirydowicz and Slaven Peles.
 
-ReSolve project would not be possible without significant contributions from (in alphabetic ortder):
+ReSolve project would not be possible without significant contributions from (in alphabetic order):
 - Maksudul Alam
 - Ryan Danehy
 - Nicholson Koukpaizan
@@ -116,9 +92,20 @@ ReSolve project would not be possible without significant contributions from (in
 - Phil Roth
 - Cameron Rutherford
 
-Development of this coede was supported by the Exascale Computing Project (ECP), Project Number: 17-SC-20-SC, a collaborative effort of two DOE organizations—the Office of Science and the National Nuclear Security Administration—responsible for the planning and preparation of a capable exascale ecosystem—including software, applications, hardware, advanced system engineering, and early testbed platforms—to support the nation's exascale computing imperative.
+Development of this code was supported by the Exascale Computing Project (ECP), Project Number: 17-SC-20-SC,
+a collaborative effort of two DOE organizations—the Office of Science and the National Nuclear Security
+Administration—responsible for the planning and preparation of a capable exascale ecosystem—including software,
+applications, hardware, advanced system engineering, and early testbed platforms—to support the nation's exascale
+computing imperative.
 
 ## License
 Copyright &copy; 2023, UT-Battelle, LLC, and Battelle Memorial Institute.
 
-ReSolve is a free software distributed under a BSD-style license. See the [LICENSE](LICENSE) and [NOTICE](NOTICE) files for details. All new contributions to ReSolve must be made under the smae licensing terms.
+ReSolve is a free software distributed under a BSD-style license. See the
+[LICENSE](LICENSE) and [NOTICE](NOTICE) files for details. All new
+contributions to ReSolve must be made under the smae licensing terms.
+
+**Please Note** If you are using ReSolve with any third party libraries linked
+in (e.g., KLU), be sure to review the respective license of the package as that
+license may have more restrictive terms than the ReSolve license.
+

